float32 ckpt_y
---
bool status